clients/admin-ui/src/features/datamap/datamap.slice.ts

```diff
@@ -68,7 +69,7 @@ const DEPRECATED_COLUMNS = [
 ];
 
 // API endpoints
-const datamapApi = baseApi.injectEndpoints({
+const datamapApi = plusApi.injectEndpoints({
   endpoints: (build) => ({
```

Comment:
this is me totally hacking on the FE code, but it _seemed_ to be sending a `GET` to the plus endpoint (`http://localhost:3000/api/v1/plus/datamap/default_organization?include_deprecated_columns=true`) when i went to `localhost:3000/datamap` so...maybe not too far off?

Reply:
ok - i think this now does what you suggested! let me know if it looks alright